ouroboros-consensus-test/src/Test/Util/Nightly.hs

```diff
@@ -35,7 +35,8 @@ instance IsOption IohkNightlyEnabled where
   defaultValue = IohkNightlyEnabled False
   parseValue = fmap IohkNightlyEnabled . safeReadBool
   optionName = pure "iohk-enable-nightly-tests"
```

Comment:
We should consider removing this option if it is not used anywhere.

Reply:
I don't see this module being used at all in our code.

Reply:
Yes, I will remove. Thanks.